### DISCUSSION TITLE
NTLM Directories doesn't run

### DISCUSSION THREAD

**Discussion by vysecurity** [projectdiscovery/nuclei-templates#3573] [Q&A] [answered]
### Issue description:
Can't seem to run NTLM Directories yaml file

```nuclei -t /root/nuclei-templates/miscellaneous/ntlm-directories.yaml```

Not sure why, I even tried -itags fuzz

**Comment by vysecurity** [comment]
The `itags` parameter is not allowing the excluded to run.

**Comment by ehsandeep** [comment, marked as answer]
`-itags fuzz,misc` is required as both tags are used in this template. @vysecurity, soon `misc` tag will be removed from default ignore list.

**Reply by vysecurity** [reply]
Oh right! I see.

**Reply by vysecurity** [reply]
Anyway to just enable all tags?
